src/em/steering/em_steering.cpp

```diff
@@ -76,14 +75,15 @@ int em_steering_t::send_client_assoc_ctrl_req_msg()
     }
 
     set_state(em_state_ctrl_configured);
+    return 0;
 }
 
 int em_steering_t::send_client_assoc_ctrl_req_msg(em_client_assoc_ctrl_req_t *assoc_ctrl)
 {
     unsigned char buff[MAX_EM_BUFF_SZ];
     char *errors[EM_MAX_TLV_MEMBERS] = {0};
     unsigned short  msg_id = em_msg_type_client_assoc_ctrl_req;
-    int len = 0;
+    unsigned int len = 0;
```

Comment:
See above. Also fixes the missed warning on line 124
```suggestion
    size_t len = 0;
```